src/PersistentOrderedSet.mo

```diff
@@ -32,11 +30,14 @@ module {
 
   /// Red-black tree of nodes with ordered set elements.
   /// Leaves are considered implicitly black.
-  public type Set<T> = {
-    #node : (Color, Set<T>, T, Set<T>);
+  public type S <T> = {
+    #node : (Color, S<T>, T, S<T>);
```

Comment:
```suggestion
  public type Tree<T> = {
    #node : (Color, Tree<T>, T, Tree<T>);
```

Reply:
You can check how I did it in the #35 
I can rename it instead of you if you want since I did it already for the Map